### DISCUSSION TITLE
Main pypi package

### DISCUSSION THREAD

**Discussion by wild-endeavor** [vaexio/vaex#1164] [Q&A]
I absolutely love the packaging structure of vaex but i'm a bit confused. the main pypi package... does that get published from the top level setup.py? It looks like it should... but it's also very clearly not: https://github.com/vaexio/vaex/blob/master/setup.py#L66

So how does the top level pypi package getting published??

**Comment by maartenbreddels** [comment]
Hi,

thanks, I love it as well, but I am hardly neutral :)
The top level setup.py is simply a wrapper to make it easy to install all packages.
The real vaex metapackage setup.py is found here: https://github.com/vaexio/vaex/blob/master/packages/vaex-meta/setup.py

Regards,

Maarten
